fcrepo-camel-common/src/main/java/org/fcrepo/camel/common/config/BasePropsConfig.java

```diff
@@ -61,6 +57,9 @@
     @Value("${fcrepo.authHost:localhost}")
     private String fcrepoAuthHost;
 
+    @Value("${fcrepo.authPort:8080}")
+    private int fcrepoAuthPort;
```

Comment:
Do we need `fcrepo.authPort`? FcrepoClient does currently not support setting a port for the authentication, we could do use the same solution in the `LDPathWrapper`?

Reply:
I'll take a look to see if we can get rid of the `authPort`. I still need to test the ldpath module to make sure it still works as expected so that will be a good opportunity to check this.

Reply:
@mikejritter Thanks for digging into this. I didn't even think of this possibility for misconfiguration - I just found it annoying, that I had to specify another property :)